scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -183,8 +183,25 @@ def configure_lnet(self):
     restart_services(self, ["lnet"])
 
 def configure_libfabric(self):
-    raise MotrError(errno.EINVAL, "libfabric not implemented\n")
+    try:
+        iface = Conf.get(self._index,
+        f'cluster>{self._server_id}')['network']['data']['private_interfaces']
+        iface = iface[0]
```

Comment:
I suppose we are going to use the first network interface in this mini-provision, right?